src/id_vl.cpp

```diff
@@ -217,13 +217,23 @@ bool sdl_initialize_window()
         title = "Blake Stone: Planet Strike (full, v1.x)";
     }
 
+#ifdef __vita__
```

Comment:
It's better to initialize appropriate variables (sdl_window_x, sdl_window_y, window_width, window_height) without changing this call.

Reply:
Good point.